src/clients/VestingData.ts

```diff
@@ -102,7 +111,10 @@ async function getVestingContractDataV2(
     contractEndsTimestamp = contractStart + contractDuration * periods
   }
 
-  return getVestingDates(contractStart, contractEndsTimestamp)
+  const released = Math.round(Number(await vestingContract.getReleased()) / 1e18)
```

Comment:
maybe even use the fn inside the `get` methods, if we only need the rounded values